packages/hasura/src/hasura.module.ts

```diff
@@ -182,6 +182,8 @@ export class HasuraModule
       if (handlers && handlers.length) {
         return Promise.all(handlers.map((x) => x.handler(evt)));
       } else {
+        console.log('key for error message', keys, typeof keys);
```

Comment:
Can you remove this or replace it with a a call through the actual logger if its needed

Reply:
@WonderPanda This was a mistake. Removed.